<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_note">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Note: In order to deactivate an alarm setting, please leave the edit field empty.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Alarm Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_alarmLow">
        <property name="text">
         <string>alarmLow</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_warnLow">
        <property name="text">
         <string>warnLow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditEditor" name="edit_warnHigh"/>
      </item>
      <item row="2" column="1">
       <widget class="LineEditEditor" name="edit_warnLow"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_warnHigh">
        <property name="text">
         <string>warnHigh</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEditEditor" name="edit_alarmLow"/>
      </item>
      <item row="0" column="1">
       <widget class="LineEditEditor" name="edit_alarmHigh"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_alarmHigh">
        <property name="text">
         <string>alarmHigh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditEditor</class>
   <extends>QLineEdit</extends>
   <header>karabogui.widgets.edits</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edit_alarmHigh</tabstop>
  <tabstop>edit_warnHigh</tabstop>
  <tabstop>edit_warnLow</tabstop>
  <tabstop>edit_alarmLow</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
